### DISCUSSION TITLE
Field condition by site language

### DISCUSSION THREAD

**Discussion by yingban** [craftcms/cms#15952] [Ideas] [answered]
Hi,

Would it be possible to have a conditional field rule based on the website language?

The context: 
We have a website with around 16 sub (regional) websites with multiple language. 
Sub website have events, and but the categories are handled in the main website.
Users from sub websites don't have direct access to the categories, so I created some fields for each language (with the config "show entries from this specific site"), which is conditioned by sites.
So for each field, I need to select 16 sites' handles.

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! Just added a “Language” condition rule to field conditions for Craft 4.13 and 5.5 (9f9e61cab032407e0f6119fce9a46a14b9c1fae5).

**Reply by brandonkelly** [reply]
Craft 4.13.0 and 5.5.0 are out with that new Language condition rule 🎉

**Comment by yingban** [comment]
Thank you @brandonkelly!
